CHANGELOG.md

```diff
@@ -5,10 +5,14 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 ## Unreleased
 
 ___Note:__ Yet to be released changes appear here._
-* `CHORE`: bump to `bpmn-js@8.2.0`
-
 ...

```

Comment:
We miss a newline here so it looks good again:

![image](https://user-images.githubusercontent.com/28307541/106865452-832b9d80-66cb-11eb-8afe-eea5b6a7fa0d.png)

Reply:
:facepalm: thanks you